packages/fxa-content-server/app/scripts/views/sign_in_token_code.js

```diff
@@ -29,6 +30,12 @@ const View = FormView.extend({
     }
   },
 
+  afterVisible() {
+    this.waitForSessionVerification(this.getAccount(), () => {
```

Comment:
I would probably add a quick comment about why we need to poll for verification here.

Reply:
Conversely, we could add a new helper function `pollForEmailBounce` or something similar, but meh.